### ISSUE TITLE
Subscriptions not updating

### ISSUE THREAD

**Issue by peterge1998** [omarroth/invidious#1035]
Youtube;
![image](https://user-images.githubusercontent.com/47355238/74610010-b3abad80-50ef-11ea-99a6-dead665301d3.png)
My invidious instance:
![image](https://user-images.githubusercontent.com/47355238/74610025-db9b1100-50ef-11ea-9ac3-d8ceb9bf6cb3.png)

I read that full_refresh might cause this. Would setting this to true solve it?
And whats the default intervall to check the rss feeds?

```
channel_threads: 1
feed_threads: 1
db:
  user: username
  password: password
  host: 172.20.0.81
  port: 5432
  dbname: invidious
full_refresh: false
https_only: false
domain:
```

**Comment by JLammer** [comment]
This is happening to my instance, as well.

Something that seems to fix (maybe temporarily) is forcing instance update, using [this shell script](https://github.com/tmiland/Invidious-Updater).

**Comment by peterge1998** [comment]
> It's very sad because this is the most used feature by me. Whithout it, I'd say Invidious is barely useful.
Yes... I run invidious in a docker container. Rebuilding and restarting doesnt help to fix that issue...

**Comment by c0ldplasma** [comment]
Same problem on docker instance

**Comment by BlipRanger** [comment]
I've been encountering this issue for a while now as well. After an update it usually works for a bit and then stops after a few days.

**Comment by JLammer** [comment]
> I've been encountering this issue for a while now as well. After an update it usually works for a bit and then stops after a few days. I did not encounter this issue until after/around release 0.20.0 iirc.

You are right.

I've forced an update 3 days ago and it work for that day. The next day it wasn't working anymore, until today it got back to work.

From what I remember, this is an issue that has been present on this project ever since 2016, but I have never found a solution. Maybe a CLI feature to force fetching and updating the feed?

For the moment, I have imported all my subscriptions to Tiny Tiny RSS and use it as a feed. Works like a charm!
